ivy/functional/ivy/linear_algebra.py

```diff
@@ -2577,9 +2594,9 @@ def trace(
     >>> x = ivy.native_array([[0, 1, 2],
     ...                       [3, 4, 5],
     ...                       [6, 7, 8]])
-    >>> y = ivy.trace(x, offset=0)
```

Comment:
Hey could you quickly explain why you're changing the offset here to 1 from 0

Reply:
just because I thought to add equal number of docstring examples for both offset=0 and offset=1 cases. So if you can observe the docstring examples you can see almost equal no of examples for both the cases. There is no specific reason for changing it but my main intention is to have equal no of examples for both these cases. 
@aparajith21 so is it one of the cause for the failing of test cases? Kindly let me know sir

Reply:
Okay, thanks for contributing, LGTM! Test failures are unrelated as you've only changed doc strings and not any code.